kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -4,7 +4,7 @@
  */
 
 const vue = require('vue');
-const VueIntl = require('vue-intl');
+// const VueIntl = require('vue-intl');
```

Comment:
It has been fixed on master - there was a version issue.

To make it build locally do the following command `npm install vue-intl@0.3.0`.